src/iuse.cpp

```diff
@@ -5140,8 +5141,9 @@ int iuse::oxytorch( player *p, item *it, bool, const tripoint & )
                ter == t_window_bars || ter == t_reb_cage ) {
         turns = to_turns<int>( 10_seconds );
     } else if( ter == t_door_metal_locked || ter == t_door_metal_c || ter == t_door_bar_c ||
-               ter == t_door_bar_locked || ter == t_door_metal_pickable || furn == f_safe_l ||
-               furn == f_gunsafe_ml || furn == f_gunsafe_mj || furn == f_gun_safe_el ) {
+               ter == t_door_bar_locked || ter == t_door_metal_pickable || ter == t_wall_metal ||
```

Comment:
I'd think doors are easier to cut through as that's enough to destroy hinges and locks. But walls may require making a human-sized hole. and that takes more time and fuel.

Reply:
Ridiculously low amounts of time spent cutting with an acetylene torch. #46257 
Is the PR that is supposed to determine the time for cutting doors and other stuff.

Reply:
sorry, missed that, mark this as resolved